### PR TITLE
scm-manager 1.47

### PR DESCRIPTION
**Pull request by sdorra**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

### CODE REVIEW

File: Formula/scm-manager.rb [hunk 1]
```diff
@@ -38,12 +38,12 @@ def install
     tools = libexec/"tools"
     tools.install resource("client")
 
-    scmCliClient = bin+"scm-cli-client"
-    scmCliClient.write <<-EOS.undent
+    scm_cli_client = bin+"scm-cli-client"
```
Comment (UniqMartin) [inline Formula/scm-manager.rb]: Please use `bin/"scm-cli-client"` instead of `bin+"scm-cli-client"`.

File: Formula/scm-manager.rb [hunk 2]
```diff
@@ -66,4 +66,8 @@ def plist; <<-EOS.undent
     </plist>
     EOS
   end
+
+  test do
+    system "#{opt_libexec}/../bin/scm-cli-client version | grep #{version}"
```
Comment (UniqMartin) [inline Formula/scm-manager.rb]: Thanks for adding a test! This should be a cleaner way to achieve a similar effect:

```ruby
assert_match version.to_s, shell_output("#{bin}/scm-cli-client version")
```

**Comment by sdorra** [comment]
done

**Comment by UniqMartin** [comment]
Merged in 244ff18cb34c240fd69eea1b35671c09a84e636a. Thank you for this contribution to Homebrew, @sdorra! :tada:

**Pull request closed without merging**